PDE/diffusion1.m

```diff
@@ -0,0 +1,139 @@
+function pde = diffusion1
+% Example PDE using the 1D Diffusion Equation. This example PDE is
+% time dependent (although not all the terms are time dependent). This
+% implies the need for an initial condition. 
+% PDE:
+% 
+% df/dt = d^2 f/dx^2
```

Comment:
Let's modify this test to have a non-unity coefficient on the diffusion term (just to make sure we are testing the non-unity value - which caught us out before)

Reply:
The coefficient in front of the diffusion term has been changed to 2 in diffusion1.m.

Reply:
So without also modifying the solution and / or the source terms, simply adding a 2 here does not allow the analytic solution and the numeric solution to agree. But also, the source terms needed to be added anyway, since in 1D the solution that was there did not satisfy the homogenous equation. I derived and implemented some source terms and it now agrees and passes the tests.